src/rodi_robot/main.py

```diff
@@ -30,7 +35,28 @@ def __init__(self):
         self.us_sensor.min_range = 0.2
         self.us_sensor.max_range = 4.0
 
+        if self.timeout > 0:
+            self._timer = threading.Timer(self.timeout, self._timeout)
+            self._set_timeout()
```

Comment:
Do we really need this timer? I was thinking on a simpler patch where the last timestamp could be stored when a message was received (i.e: rospy.Time.now() ) and check for the delta on the rospy main loop.